config.json

```
@@ -56,6 +56,18 @@
           "dates"
         ]
       },
+      {
+        "slug": "beer-song",
+        "name": "99 Bottles of Beer",
```

I've checked, and all other tracks use `Beer Song` as the name, so it's probably best to be consistent:

```suggestion
        "name": "Beer Song",
```